packages/dialog/stories/dialog-base.stories.ts

```diff
@@ -10,33 +10,31 @@ governing permissions and limitations under the License.
 */
 
 import { html, TemplateResult } from '@spectrum-web-components/base';
+import '@spectrum-web-components/button/sp-button.js';
+import '@spectrum-web-components/checkbox/sp-checkbox.js';
 import '@spectrum-web-components/dialog/sp-dialog-base.js';
 import '@spectrum-web-components/dialog/sp-dialog.js';
-import '@spectrum-web-components/button/sp-button.js';
+import { trigger } from '@spectrum-web-components/overlay';
 import '@spectrum-web-components/overlay/sp-overlay.js';
-import '@spectrum-web-components/checkbox/sp-checkbox.js';
 import { alertDestructive } from './dialog.stories.js';
 import { portrait } from './images.js';
 import { disabledButtonDecorator } from './index.js';
 
+const withOverlayDecorator = (story: () => TemplateResult): TemplateResult => {
```

Comment:
This decorator needs to be initialized. This will create problems in VRT tests

Reply:
Maybe I'm missing something here but I'm not sure what you mean by initializing it. I extracted the function as it was previously defined for all the stories from this file, to be able to add it only where needed.
Is this related to the VRTs saying that this PR removes the disabledButton VRT ([link](https://24258a3f64459344914e0775a9297752--spectrum-web-components.netlify.app/review/#DialogBaseStories/disabledButton.png))?

Reply:
I see you are trying to add `withOverlayDecorator` decorator with every story! Instead of adding this as story.decorator = [withOverlayDecorator] you can initialise this as 
```
export default { 
....
decorators:[withOverlayDecorator()]
}
```
But this is not something that will trigger the issue! In the mean time I see some flaky tests have emerged and a redundant hash is there. Can you please updated the hash and surface up the new baseline for slotted and re-trigger the runs.

Reply:
That decorator is used with every story _except the one that this PR adds_, that's why I extracted it from the default export, I don't want it applied in said story. That story is created different to emphasize the lazy loaded usage of the component.
As for the flaky tests, I will update the hash right away.

Reply:
OK this is failing coz when you add `disabledButton.decorators = [withOverlayDecorator, disabledButtonDecorator]`, the decorators are not getting ready. Can you try adding them separately instead?
```js
disabledButton.decorators = [withOverlayDecorator];
disabledButton.decorators = [disabledButtonDecorator];
```